Mage.Sets/src/mage/cards/m/MaskwoodNexus.java

```diff
@@ -124,24 +124,18 @@ public boolean apply(Layer layer, SubLayer sublayer, Ability source, Game game)
                     && stackObject.isControlledBy(source.getControllerId())
                     && stackObject.isCreature()) {
                 Card card = ((Spell) stackObject).getCard();
-                card.setIsAllCreatureTypes(true);
+                game.getState().getCreateMageObjectAttribute(card, game).getSubtype().setIsAllCreatureTypes(true);
```

Comment:
Why you don't use `card.setIsAllCreatureTypes(true, game)` here?

Reply:
because the `getSubType` method doesn't necessarily get the `MageObjectAttribute`

Reply:
maybe it should? I dunno

Reply:
`setIsAllCreatureTypes(boolean, game)` must use `getCreateMageObjectAttribute`